sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapExamples.cs

```diff
@@ -123,5 +123,124 @@ public void InterestRateSwapPortfolioCashFlowsExample(ModelSelection.ModelEnum m
+        [TestCase(ModelSelection.ModelEnum.Discounting)]
+        public void LifeCycleManagementForInterestRateSwap(ModelSelection.ModelEnum model)
+        {
+            // CREATE an InterestRateSwap
+            var interestRateSwap = InstrumentExamples.CreateExampleInterestRateSwap(type: "Vanilla");
+            // Pick out the currency for future reference.
+            var fixedLeg = (FixedLeg) interestRateSwap.Legs
+                .First(leg => leg.InstrumentType == LusidInstrument.InstrumentTypeEnum.FixedLeg);
+            var currency = fixedLeg.LegDefinition.Conventions.Currency;
+
+            // CREATE wide enough window to pick up all cashflows associated to the InterestRateSwap
+            var windowStart = interestRateSwap.StartDate.AddMonths(-1);
+            var windowEnd = interestRateSwap.MaturityDate.AddMonths(1);
+
+            // CREATE portfolio and add instrument to the portfolio
+            var scope = Guid.NewGuid().ToString();
+            var (instrumentID, portfolioCode) = CreatePortfolioAndInstrument(scope, interestRateSwap);
+
+            // Populate stores with required market data.
+            CreateAndUpsertMarketDataToLusid(scope, model, interestRateSwap);
+
+            // CREATE recipe to price the portfolio with
+            var recipeCode = CreateAndUpsertRecipe(scope, model, windowValuationOnInstrumentStartEnd: true);
+
+            // We expect that the PV of the InterestRateSwap should be zero after the maturity.
+            // CREATE valuation request for this portfolio consisting of the InterestRateSwap,
+            // with valuation dates a few days before, day of and a few days after the instrument expiration.
+            var valuationRequest = TestDataUtilities.CreateValuationRequest(
+                scope,
+                portfolioCode,
+                recipeCode,
+                effectiveAt: interestRateSwap.MaturityDate.AddDays(2).AddMilliseconds(1),
+                effectiveFrom: interestRateSwap.MaturityDate.AddDays(-2).AddMilliseconds(1)); // TODO ANA-910: should I mention the millisecond issue?
+
+            // CALL GetValuation before upserting back the cash flows. We check
+            // (1) there is no cash holdings in the portfolio prior to expiration
+            // (2) that when the InterestRateSwap has expired, the PV is zero.
+            var valuationBeforeAndAfterExpirationInterestRateSwap = _aggregationApi.GetValuation(valuationRequest);
+            TestDataUtilities.CheckNoCashPositionsInValuationResults(
+                valuationBeforeAndAfterExpirationInterestRateSwap,
+                currency);
+            TestDataUtilities.CheckNonZeroPvBeforeMaturityAndZeroAfter(
+                valuationBeforeAndAfterExpirationInterestRateSwap,
+                interestRateSwap.MaturityDate);
+
+            // GET all upsertable cashflows (transactions) for the InterestRateSwap.
+            // EffectiveAt after maturity so we have all the data.
```

Comment:
suggest "data" -> "reset quotes" or alike

Reply:
changed to "so we have all the cash flows"